public/app/features/plugins/admin/api.ts

```diff
@@ -20,6 +20,7 @@ export async function getPluginDetails(id: string): Promise<CatalogPluginDetails
   const dependencies = local?.dependencies || remote?.json?.dependencies;
 
   return {
+    isDeprecated: remote?.status === 'deprecated',
```

Comment:
nit: Create an enum here instead of using a hardcoded value.

Reply:
Updated in https://github.com/grafana/grafana/pull/74598/commits/7568f765183a88119bdec8cb5e48e8bb96d9daa9.